manifests/cf-manifest/manifest/020-cf-properties.yml

```diff
@@ -613,3 +613,8 @@ properties:
 
   garden:
     log_level: error
+  racoon:
+    certificate_authority_cert: (( grab secrets.bosh_ca_cert ))
+    certificate_authority_private_key: (( grab secrets.bosh_ca_key ))
```

Comment:
Yes, that's reasonable. However we should do that for any other private key that we've got currently in our manifest.
```
cf-manifest/manifest$ grep -R _key .
./020-cf-properties.yml:    host_key_fingerprint: (( grab secrets.ssh_proxy_host_key.public_fingerprint ))
./020-cf-properties.yml:    agent_key: (( grab secrets.consul_agent_key ))
./020-cf-properties.yml:    encrypt_keys: (( grab secrets.consul_encrypt_keys ))
./020-cf-properties.yml:    server_key: (( grab secrets.consul_server_key ))
./020-cf-properties.yml:    client_key: ""
./020-cf-properties.yml:    peer_key: ""
./020-cf-properties.yml:    server_key: ""
./020-cf-properties.yml:      client_key: (( grab properties.etcd.client_key ))
./020-cf-properties.yml:      server_key: ~
./020-cf-properties.yml:      client_key: (( grab properties.etcd.client_key ))
./020-cf-properties.yml:      client_key: ~
./020-cf-properties.yml:      client_key: (( grab properties.etcd.client_key ))
./020-cf-properties.yml:      client_key: (( grab properties.etcd.client_key ))
./020-cf-properties.yml:    db_encryption_key: (( grab secrets.cc_db_encryption_key ))
./020-cf-properties.yml:      resource_directory_key: (( concat meta.environment "-cf-resources" ))
./020-cf-properties.yml:      app_package_directory_key: (( concat meta.environment "-cf-packages" ))
./020-cf-properties.yml:      droplet_directory_key: (( concat meta.environment "-cf-droplets" ))
./020-cf-properties.yml:      license_key: ~
./020-cf-properties.yml:      buildpack_directory_key: (( concat meta.environment "-cf-buildpacks" ))
./020-cf-properties.yml:      verification_key: (( grab secrets.uaa_jwt_verification_key ))
./020-cf-properties.yml:      signing_key: (( grab secrets.uaa_jwt_signing_key ))
./020-cf-properties.yml:        client_key: (( grab secrets.bbs_client_key  ))
./020-cf-properties.yml:        client_key: (( grab secrets.bbs_client_key  ))
./020-cf-properties.yml:      active_key_label: keylabel1
./020-cf-properties.yml:      encryption_keys:
./020-cf-properties.yml:      server_key: (( grab secrets.bbs_server_key ))
./020-cf-properties.yml:        client_key: (( grab secrets.consul_agent_key ))
./020-cf-properties.yml:        client_key: (( grab secrets.bbs_client_key ))
./020-cf-properties.yml:        client_key: (( grab secrets.bbs_client_key ))
./020-cf-properties.yml:        client_key: (( grab secrets.bbs_client_key ))
./020-cf-properties.yml:      host_key: (( grab secrets.ssh_proxy_host_key.private_key ))
./020-cf-properties.yml:        client_key: (( grab secrets.bbs_client_key  ))
./020-cf-properties.yml:        client_key: (( grab secrets.bbs_client_key ))
./020-cf-properties.yml:        client_key: (( grab secrets.bbs_client_key ))
./050-rds-broker.yml:        aws_access_key_id: ""
./050-rds-broker.yml:        aws_secret_access_key: ""
./050-rds-broker.yml:        state_encryption_key: (( grab secrets.rds_broker_state_encryption_key ))
```